alfresco-googledrive-repo-enterprise/src/main/docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alfresco/alfresco-content-repository:6.2.0
 
-ARG USERNAME=alfresco
+ARG THEUSERNAME=alfresco
```

Comment:
Why did this need to be renamed ? 

Note: IIRC, this pattern of using USERNAME is used by other Dockerfile (in various other projects/repos).